config.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "github.com/traceableai/goagent/config";
```

Comment:
I'm not sure if we want to put this into the goagent package, or have it's own package such as `github.com/hypertrace/agent/config`  or maybe even just the repo name `github.com/hypertrace/agent-config`

I can think of other go consumers of this which may not be the goagent.

Reply:
My bad, I should have changed this when moved to this repo. Let me fix it.